multiversx_sdk_core/testutils/wallets.py

```diff
@@ -10,7 +10,7 @@
 carol_pem = Path(__file__).parent / "testwallets" / "carol.pem"
 
 
-def load_wallets():
+def load_wallets() -> dict[str, UserPEM]:
```

Comment:
Or uppercase `Dict[...]`, for consistency.

Reply:
Indeed. Changed!

Reply:
Done!